File: sample-chat/src/main/java/com/quickblox/sample/chat/ui/activity/AppInfoActivity.java
package com.quickblox.sample.chat.ui.activity;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import com.quickblox.sample.chat.BuildConfig;
import com.quickblox.sample.chat.R;
import com.quickblox.sample.core.utils.VersionUtils;

/**
 * Created by pavlov egor on 11/1/18.
 */

public class AppInfoActivity extends BaseActivity {

    private TextView tvAppVersion;
    private TextView tvSDKVersion;
    private TextView tvAppID;
    private TextView tvAuthKey;
    private TextView tvAuthSecret;
    private TextView tvAccountKey;
    private TextView tvApiDomain;
    private TextView tvChatDomain;

    public static void start(Context context) {
        Intent intent = new Intent(context, AppInfoActivity.class);
        context.startActivity(intent);
    }

    @Override
    protected View getSnackbarAnchorView() {
        return null;
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_appinfo);

        initUI();
        fillUI();
    }

    private void initUI() {
        actionBar.setDisplayHomeAsUpEnabled(true);
        tvAppVersion = _findViewById(R.id.tv_app_version);
        tvSDKVersion = _findViewById(R.id.tv_sdk_version);
        tvAppID = _findViewById(R.id.tv_app_id);
        tvAuthKey = _findViewById(R.id.tv_auth_key);
        tvAuthSecret = _findViewById(R.id.tv_auth_secret);
        tvAccountKey = _findViewById(R.id.tv_account_key);
        tvApiDomain = _findViewById(R.id.tv_api_domain);
        tvChatDomain = _findViewById(R.id.tv_chat_domain);
    }

    public void fillUI() {
        tvAppVersion.setText(VersionUtils.getAppVersionName());
        tvSDKVersion.setText(BuildConfig.qbSdkVer);
        tvAppID.setText(VersionUtils.getAppId());
        tvAuthKey.setText(VersionUtils.getAuthorizationKey());
        tvAuthSecret.setText(VersionUtils.getAuthorizationSecret());
        tvAccountKey.setText(VersionUtils.getAccountKey());
        tvApiDomain.setText(VersionUtils.getApiDomain());
        tvChatDomain.setText(VersionUtils.getChatDomain());
    }
}

File: sample-chat-kotlin/app/src/main/java/com/quickblox/sample/chat/kotlin/ui/activity/AppInfoActivity.kt
package com.quickblox.sample.chat.kotlin.ui.activity

import android.content.Context
import android.content.Intent
import android.graphics.Color
import android.os.Bundle
import android.text.Spannable
import android.text.SpannableString
import android.text.style.ForegroundColorSpan
import android.view.View
import android.widget.TextView
import com.quickblox.auth.session.QBSettings
import com.quickblox.sample.chat.kotlin.BuildConfig
import com.quickblox.sample.chat.kotlin.R


class AppInfoActivity : BaseActivity() {
    private lateinit var appVersionTextView: TextView
    private lateinit var sdkVersionTextView: TextView
    private lateinit var appIDTextView: TextView
    private lateinit var authKeyTextView: TextView
    private lateinit var authSecretTextView: TextView
    private lateinit var accountKeyTextView: TextView
    private lateinit var apiDomainTextView: TextView
    private lateinit var chatDomainTextView: TextView
    private lateinit var appQAVersionTextView: TextView

    companion object {
        fun start(context: Context) = context.startActivity(Intent(context, AppInfoActivity::class.java))
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_appinfo)
        initUI()
        fillUI()
    }

    private fun initUI() {
        appVersionTextView = findViewById(R.id.tv_app_version)
        sdkVersionTextView = findViewById(R.id.tv_sdk_version)
        appIDTextView = findViewById(R.id.tv_app_id)
        authKeyTextView = findViewById(R.id.tv_auth_key)
        authSecretTextView = findViewById(R.id.tv_auth_secret)
        accountKeyTextView = findViewById(R.id.tv_account_key)
        apiDomainTextView = findViewById(R.id.tv_api_domain)
        chatDomainTextView = findViewById(R.id.tv_chat_domain)
        appQAVersionTextView = findViewById(R.id.tv_qa_version)
    }

    private fun fillUI() {
        supportActionBar?.setDisplayHomeAsUpEnabled(true)
        supportActionBar?.title = getString(R.string.appinfo_title)
        appVersionTextView.text = BuildConfig.VERSION_NAME
        sdkVersionTextView.text = com.quickblox.BuildConfig.VERSION_NAME
        appIDTextView.text = QBSettings.getInstance().applicationId
        authKeyTextView.text = QBSettings.getInstance().authorizationKey
        authSecretTextView.text = QBSettings.getInstance().authorizationSecret
        accountKeyTextView.text = QBSettings.getInstance().accountKey
        apiDomainTextView.text = QBSettings.getInstance().serverApiDomain
        chatDomainTextView.text = QBSettings.getInstance().chatEndpoint

        if (BuildConfig.IS_QA) {
            val appVersion = BuildConfig.VERSION_NAME
            val versionQACode = BuildConfig.VERSION_QA_CODE.toString()
            val qaVersion = "$appVersion.$versionQACode"
            val spannable = SpannableString(qaVersion)
            spannable.setSpan(ForegroundColorSpan(Color.RED), appVersion.length + 1,
                    qaVersion.length, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE)
            appQAVersionTextView.setText(spannable, TextView.BufferType.SPANNABLE)
            appQAVersionTextView.visibility = View.VISIBLE

            findViewById<View>(R.id.text_qa_version_title).visibility = View.VISIBLE
        }
    }
}